components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/internal/constant/Constants.java

```diff
@@ -52,6 +52,8 @@ public class Constants {
 
     public static final String SESSION_ID = "sessionId";
 
+    public static final String PRE_DELETE_USER_USER_ID_FOR_WEB_SUB_HUB = "preDeleteUserUserIdForWebSubHub";
```

Comment:
1. why we need to have a websubhub specific constant ?
2. why we say 'UserUserId'

Reply:
This is an existing constant used for asgardeo-extensions. We can rename this constant once our services are on live and asgardeo-extention is deprecated. Until that I believe it is better to keep the constant. WDYT ?

Reply:
Fixed 8d8f0ebd9e807c0ac81c21e5a19eec711c59195f